tests/serialize/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Yeah I can see that. I would stay away from building tools if you can since the building is not very complex and maintaining the plugins and scripts for those is too much of a hassle imo, I would rather temporary place the scripts on the package.json. 
I've rewritten a couple of the tests in mocha for now and made data.js into a class that return promises, they should all work async. I'm writing all in typescript cause strong typing saves me some time, but I can just transpile them to es6 or directly in es5 if it's a problem.